### ISSUE TITLE
Can a variable rely on census variables from multiple sources?

### ISSUE THREAD

**Issue by mgraber** [NYCPlanning/db-factfinder#114]
The method `download_e_m` in the `Download` class allows for census variables from multiple sources.
https://github.com/NYCPlanning/db-factfinder/blob/21af4a60c7b7d91dcf68556930a95e6b8c74207e/factfinder/download.py#L86-L113

This relies on the now deprecated method `create_census_variables` method in the `Variable` class. The new method, `census_variables`, does not take a list of variables as an input, instead using the `census_variable` property from metadata. As such, it doesn't allow for pre-filtering based on source.

Our metadata, however, also has a "source" property, suggesting that each variable pulls census variables from a single source. Can we remove the multi-source functionality from `Download.download_e_m`?

**Comment by SPTKL** [comment]
See example of mixed source variables:
```json
{
    "base_variable": "hovacrt",
    "census_variable": [
        "B25004_004",
        "B25004_005",
        "DP04_0046"
    ],
    "domain": "housing",
    "pff_variable": "hovacu",
    "rounding": 0,
    "source": null
}, 
{
    "base_variable": "rntvacrt",
    "census_variable": [
        "B25004_002",
        "B25004_003",
        "DP04_0047"
    ],
    "domain": "housing",
    "pff_variable": "rntvacu",
    "rounding": 0,
    "source": null
}
```

**Comment by mgraber** [comment]
@SPTKL We need to either revert or modify merged PR #110 then.

**Comment by mgraber** [comment]
PR #110 is reverted in refactor/geography
